src/s1reader/s1_burst_slc.py

```diff
@@ -421,30 +446,29 @@ def bistatic_delay(self, xstep=1, ystep=1):
 
         Parameters
         -------
-        xstep : int
-           spacing along x direction (range direction) in units of pixels
-
-        ystep : int
-           spacing along y direction (azimuth direction) in units of pixels
+        range_step : int
+            Spacing along x/range direction [meters]
+        az_step : int
+            Spacing along y/azimuth direction [seconds]
+        use_radar_coords : bool
```

Comment:
was this meant to be added as an option too? Right now there's just range/az step in the function

Reply:
I forgot to remove the above from the docstrings after modifying function.